sdk/include/opentelemetry/sdk/metrics/view/view.h

```diff
@@ -21,26 +22,22 @@ class View
 {
 public:
   View(const std::string &name,
-       const std::string &description = "",
-       std::unique_ptr<opentelemetry::sdk::metrics::Aggregation> aggregation =
-           std::unique_ptr<opentelemetry::sdk::metrics::Aggregation>(new DropAggregation()),
+       const std::string &description   = "",
+       AggregationType aggregation_type = AggregationType::kDrop,
        std::unique_ptr<opentelemetry::sdk::metrics::AttributesProcessor> attributes_processor =
            std::unique_ptr<opentelemetry::sdk::metrics::AttributesProcessor>(
                new opentelemetry::sdk::metrics::DefaultAttributesProcessor()))
       : name_(name),
         description_(description),
-        aggregation_{std::move(aggregation)},
+        aggregation_type_{aggregation_type},
         attributes_processor_{std::move(attributes_processor)}
   {}
 
   virtual std::string GetName() const noexcept { return name_; }
 
   virtual std::string GetDescription() const noexcept { return description_; }
 
-  virtual const opentelemetry::sdk::metrics::Aggregation &GetAggregation() const noexcept
-  {
-    return *aggregation_.get();
-  }
+  virtual const AggregationType &GetAggregationType() const noexcept { return aggregation_type_; }
```

Comment:
Why do we need to return `const&` here?

Reply:
Good point. Earlier, this method was returning `Aggregation` object and so was `const &`. . It is not required now, so removed.